pkg/input/golden_test/tfplan/sensitive.json

```diff
@@ -0,0 +1,68 @@
+{
+  "format": "",
+  "format_version": "",
+  "input_type": "tf_plan",
+  "environment_provider": "iac",
+  "meta": {
+    "filepath": "golden_test/tfplan/sensitive/plan.json"
+  },
+  "resources": {
+    "aws_db_instance": {
+      "aws_db_instance.default": {
+        "id": "aws_db_instance.default",
+        "resource_type": "aws_db_instance",
+        "namespace": "golden_test/tfplan/sensitive/plan.json",
+        "meta": {
+          "region": "us-east-1",
+          "terraform": {
+            "provider_config": {
+              "region": "us-east-1"
+            }
+          },
+          "tfplan": {
+            "resource_actions": [
+              "create"
+            ]
+          }
+        },
+        "attributes": {
+          "allocated_storage": 10,
+          "allow_major_version_upgrade": null,
+          "auto_minor_version_upgrade": true,
+          "copy_tags_to_snapshot": false,
+          "customer_owned_ip_enabled": null,
+          "db_name": "mydb",
+          "delete_automated_backups": true,
+          "deletion_protection": null,
+          "domain": null,
+          "domain_iam_role_name": null,
+          "enabled_cloudwatch_logs_exports": null,
+          "engine": "mysql",
+          "engine_version": "5.7",
+          "final_snapshot_identifier": null,
+          "iam_database_authentication_enabled": null,
+          "instance_class": "db.t3.micro",
+          "iops": null,
+          "max_allocated_storage": null,
+          "monitoring_interval": 0,
+          "parameter_group_name": "default.mysql5.7",
+          "password": null,
```

Comment:
Is it possible to have a non-null value here? I think it might be important for us to be able to distinguish that a property is set. It could be something like a bunch of `*`s or we could try doing what CircleCI (and I think GitHub?) do where they redact everything after the first few characters.

Reply:
Yes, since we're in the schemas anyway we could do that whenever it's sensitive & a string.  Using `null` as a fallback for other rare cases probably still makes sense though?

Reply:
I've replaced these by `******`, I felt like we shouldn't leak the length nor some characters of the password if the goal is to strip out sensitive info.